src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationService.cs

```diff
@@ -69,6 +118,8 @@ public async Task<CreateApplicationRequest> BuildCreateApplicationRequest(string
 
     public interface IApplicationService
```

Comment:
Can we split the interface out into a separate file to be consistent with everywhere else?

Reply:
Done